.github/workflows/main.yml

```diff
@@ -63,6 +63,9 @@ jobs:
       - name: Cargo build
         run: cargo build --workspace --features bundled
 
+      - name: Cargo clippy
+        run: cargo clippy --workspace --features bundled -- -D warnings
+
```

Comment:
It might be nice to use the [clippy-check](https://github.com/actions-rs/clippy-check) action so that lints are shown inline in the GitHub commits.

Reply:
`clippy-check`, like all of the `actions-rs` stuff is unmaintained. (and likely triggers deprecation warnings due to old node js versions, like the bulk of the `actions-rs` actions).

Reply:
> `clippy-check`, like all of the `actions-rs` stuff is unmaintained. (and likely triggers deprecation warnings due to old node js versions, like the bulk of the `actions-rs` actions).

Oh, good to know. Thanks.